File: app/src/main/java/fobo66/exchangecourcesbelarus/util/comparators/USDSellComparator.java
package fobo66.exchangecourcesbelarus.util.comparators;

import fobo66.exchangecourcesbelarus.models.Currency;

public class USDSellComparator implements CurrencyComparator {
    @Override
    public int compare(Currency lhs, Currency rhs) {
        return rhs.usdSell.compareToIgnoreCase(lhs.usdSell);
    }
}

File: app/src/main/java/fobo66/exchangecourcesbelarus/util/comparators/UsdSellComparator.kt
package fobo66.exchangecourcesbelarus.util.comparators

import fobo66.exchangecourcesbelarus.entities.Currency

class UsdSellComparator : Comparator<Currency> {
  override fun compare(lhs: Currency, rhs: Currency): Int {
    return rhs.usdSell.compareTo(lhs.usdSell, ignoreCase = true)
  }
}
